docs/db_overview.md

```diff
@@ -0,0 +1,55 @@
+
+
+|Master Table Parameter | Purpose|
+|:-: | :- |
+|unique_id | A unique ID generated for each entry of the database.|
+|experiment_name | A name given to the experiment through the experiment's configuration file. If not specified, default value is set to "default name".|
+|experiment_description | A set of text provided to describe the function of the experiment. This value is configurable in the experiment's configuration file. If not specified, default value is set to "default description".|
+|experiment_id | A value distinguishing the specific experiment in question. This value utilizes the python Universal Unique Identifier module. If not specified in the experiment's configuration file, it will be automatically generated through the python module.|
+|participant_id | A value specifying a particular user within an experiment (perfect for keeping track of more than one participant in an experiment). If not specified in the experiment's configuration file, it will be empty in the Database.|
+| extra_metadata | Any special meta data specified in the configuration file under the section "Metadata". This is automatically generated into a JSON file so it may be serialized/deserialized easier at a later time period. If no metadata is specified in the configuration file, this field will be generated from default values. |
+
+
+- **replay_data Table**: The replay data table is essential for replaying experiments. It contains entries from each step of an experiment's trial so the replay functionality can work properly. The parameters are described as followed:
+
+|Replay Data Parameter | Purpose |
+|:-: | :- |
+|unique_id | A unique ID generated for each entry of the database. |
+|timestamp | An identifier that specifies a timestamp of the specific entry. This parameter helps in providing a chronological order for the replay to replay events. |
+|message_type | This is the type of message generated by the server in this entry. These message types include _setup_, _ask_, and _tell_.|
+|message_contents | Contains the raw data in the message_content field of the message. This is stored as a BLOB in the database.|
+|extra_info | Extra information pertaining to the replay data. This is stored as a Python pickle for serialization/deserialization. This may change in the future for more preferred methods. |
+|master_table_id | This is a reference key to the original entry in the master table's unique ID. |
+
+- **strat_data table**: The strat_data table contains information pertaining to the choices of which strat is used during an experiment and stores information about such strats. The parameters are described as followed:
+
+|Strat Table Parameter | Purpose |
+| :-: | :- |
+|unique_id | A unique ID generated for each entry of the database. |
+|timestamp | An identifier that specifies a timestamp of the specific entry. This parameter helps in providing a chronological order in the viewing of the strats history.|
```

Comment:
Is the timestamp from when the strategy object is first created? When the strategy generates its first trial? When the strategy finishes?